core/capabilities/ccip/oraclecreator/plugin.go

```diff
@@ -218,6 +220,18 @@ func (i *pluginOracleCreator) Create(ctx context.Context, donID uint32, config c
 	return newWrappedOracle(oracle, closers), nil
 }
 
+func encodeOffRampAddr(addr []byte, chainFamily string) string {
+	var offRampAddr string
+	switch chainFamily {
+	case relay.NetworkEVM:
+		offRampAddr = hexutil.Encode(addr)
+	case relay.NetworkSolana:
+		offRampAddr = solana.PublicKeyFromBytes(addr).String()
+	}
```

Comment:
I'd prefer to error or panic here if encounter an unknown family. That will make it explicit to developers to support a new family here too, instead of silently returning null string.